pkg/gofr/cmd.go

```diff
@@ -1,70 +1,173 @@
 package gofr
 
 import (
+	"fmt"
 	"os"
 	"regexp"
-
-	"gofr.dev/pkg/gofr/container"
+	"strings"
 
 	cmd2 "gofr.dev/pkg/gofr/cmd"
+	"gofr.dev/pkg/gofr/container"
 )
 
 type cmd struct {
 	routes []route
 }
 
 type route struct {
-	pattern string
-	handler Handler
+	pattern     string
+	handler     Handler
+	description string
+	help        string
+	fullPattern string
```

Comment:
What is the use case for `fullpatern` field?

Reply:
Hi @vipul-rawat ,
The fullpattern also matches the same way as pattern but it will be like -h for pattern and --help for fullpattern.

Reply:
But I believe this would be fixed in your PR @KedarisettiSreeVamsi, so not required as part of this PR and issue